source/opt/common_uniform_elim_pass.cpp

```diff
@@ -117,7 +116,8 @@ ir::Instruction* CommonUniformElimPass::GetPtr(
     ptrInst = def_use_mgr_->GetDef(*varId);
   }
   ir::Instruction* varInst = ptrInst;
-  while (varInst->opcode() != SpvOpVariable) {
+  while (varInst->opcode() != SpvOpVariable &&
```

Comment:
Need to update the description of GetPtr in common_uniform_elim_pass.h, including the fact that the varId parameter is now badly named.  Maybe it should be objId?